lib/route.js

```diff
@@ -155,31 +155,31 @@ function buildRouting (options) {
       if (path === '/' && prefix.length > 0) {
         switch (opts.prefixTrailingSlash) {
           case 'slash':
-            afterRouteAdded.call(this, path, notHandledErr, done)
+            afterRouteAdded.call(this, { path }, notHandledErr, done)
             break
           case 'no-slash':
-            afterRouteAdded.call(this, '', notHandledErr, done)
+            afterRouteAdded.call(this, { path: '' }, notHandledErr, done)
             break
           case 'both':
           default:
-            afterRouteAdded.call(this, '', notHandledErr, done)
+            afterRouteAdded.call(this, { path: '' }, notHandledErr, done)
             // If ignoreTrailingSlash is set to true we need to add only the '' route to prevent adding an incomplete one.
             if (ignoreTrailingSlash !== true) {
-              afterRouteAdded.call(this, path, notHandledErr, done)
+              afterRouteAdded.call(this, { path, prefixing: true }, notHandledErr, done)
             }
         }
       } else if (path[0] === '/' && prefix.endsWith('/')) {
         // Ensure that '/prefix/' + '/route' gets registered as '/prefix/route'
-        afterRouteAdded.call(this, path.slice(1), notHandledErr, done)
+        afterRouteAdded.call(this, { path: path.slice(1) }, notHandledErr, done)
       } else {
-        afterRouteAdded.call(this, path, notHandledErr, done)
+        afterRouteAdded.call(this, { path }, notHandledErr, done)
       }
     })
 
     // chainable api
     return this
 
-    function afterRouteAdded (path, notHandledErr, done) {
+    function afterRouteAdded ({ path, prefixing = false }, notHandledErr, done) {
```

Comment:
Can we add a jsdoc block here to describe what this function is, its purpose, and the parameters?